sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -4039,7 +4039,7 @@ pi_result cuda_piEnqueueMemImageFill(pi_queue command_queue, pi_mem image,
 ///
 pi_result cuda_piEnqueueMemBufferMap(pi_queue command_queue, pi_mem buffer,
                                      pi_bool blocking_map,
-                                     cl_map_flags map_flags, size_t offset,
+                                     pi_mem_flags map_flags, size_t offset,
```

Comment:
```suggestion
                                     pi_map_flags map_flags, size_t offset,
```

Reply:
And other places too

Reply:
Thanks again. They were all wrong. I went through and audited the changes and re-fixed them.